### ISSUE TITLE
Alert when pasting SRM target proteins (Step 3, 01-Create-...)

### ISSUE THREAD

**Issue by grace-ac** [RobertsLab/Paper-DNR-Geoduck-Proteomics#5]
Should I remove? 

![capture_geoduck](https://user-images.githubusercontent.com/14934314/31294726-e4f1d872-aa90-11e7-853c-2547e65828f9.PNG)

**Comment by sr320** [comment]
And also modify protocol with answer.

**Comment by laurahspencer** [comment]
That's strange, I just tried creating a new Skyline-daily project w/ settings, background proteome, and when importing targets I didn't get that error.  does your Skyline document have all the proteins/peptides that mine has?  Open my Skyline project (see screenshot below for location on the Windows computer), and yours at the same time, and compare.  Note that my project is likely further along in the protocol than yours, where I've removed all peptides not targeted for analysis in this run: 

![image](https://user-images.githubusercontent.com/17264765/31403949-8dfd12f8-adaf-11e7-9b77-28f38ae18b97.png)
